src/Math/babylon.math.ts

```diff
@@ -1914,9 +1914,9 @@
          * @returns the current updated Vector3
          */
         public maximizeInPlaceFromFloats(x: number, y: number, z: number): Vector3 {
-            this.x = Math.max(this.x, x);
-            this.y = Math.max(this.y, y);
-            this.z = Math.max(this.z, z);
+            if (x > this.x) this.x = x;
+            if (y > this.y) this.y = y;
+            if (z > this.z) this.z = z;

```

Comment:
changes in this files are not really related to the PR subject.

I replaced `Math.min` and `Math.max` with if condition as it used to be before my other PR https://github.com/BabylonJS/Babylon.js/pull/5217.